test/modules/Modules.Test/TestResultCoordinator/Reports/DirectMethod/DirectMethodConnectivityReportDataWithSenderSourceOnly.cs

```diff
@@ -84,7 +84,7 @@ public class DirectMethodConnectivityReportDataWithSenderSourceOnly
                 {
                     // NetworkOnFailure test
                     Enumerable.Range(1, 7).Select(v => (ulong)v),
-                    new List<HttpStatusCode> { HttpStatusCode.InternalServerError, HttpStatusCode.OK, HttpStatusCode.OK, HttpStatusCode.NotFound, HttpStatusCode.OK, HttpStatusCode.OK, HttpStatusCode.OK },
+                    new List<HttpStatusCode> { HttpStatusCode.InternalServerError, HttpStatusCode.InternalServerError, HttpStatusCode.InternalServerError, HttpStatusCode.NotFound, HttpStatusCode.OK, HttpStatusCode.OK, HttpStatusCode.OK },
```

Comment:
I wanted to keep the test functioning as expected, ignoring any tolerances. Therefore I had to exceed the tolerance (by adding more errors) so that it would surpass the new threshold.